indigo_api/timeline.py

```diff
@@ -1,12 +1,22 @@
 from dataclasses import dataclass, field
 
+from django.db.models import TextChoices
 from django.utils.formats import date_format
 from django.utils.translation import gettext as _
 
 
+class TimelineEventType(TextChoices):
```

Comment:
what does TextChoices do? (should we be using it on Task for verbs and codes?)

Reply:
The sole motivation for this is it means the API documentation shows what the possible values are.

So we don't really need it for Tasks.